dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/CDIContainerImpl.java

```diff
@@ -639,16 +635,17 @@ public String getCurrentApplicationContextID() {
      * @return
      * @throws CDIException
      */
+    @SuppressWarnings("deprecation") //Until nobody uses this deprecated interface we still need to read it
```

Comment:
Not clear which deprecated interface is being referred to.

Reply:
In the IDE, it's more obvious because `WebSphereCDIExtension` is struck through.

The comment should either mention `WebSphereCDIExtension` or be removed.